demo/client/backend/mock/server.go

```diff
@@ -59,7 +59,7 @@ func main() {
 	flag.Parse()
 
 	if flagDebug {
-		logger.SetLevel(shim.LogDebug)
+		//logger.SetLevel(shim.LogDebug)
```

Comment:
was this breaking something?

Reply:
Seems that the flogger logger does not support `SetLevel()`. I added a todo

Reply:
looking at docu, i guess you have to provide a complete fabric logging spec and set it via https://godoc.org/github.com/hyperledger/fabric/common/flogging#ActivateSpec?  Maybe flogging.ActivateSpec("debug") or logger.ActivateSpec("debug") might be a quick hack to get all logging?  That said, doesn't have to be in this PR

Reply:
updated #248 correspondingly